mono/metadata/mono-private-unstable.h

```diff
@@ -16,10 +16,15 @@
 
 typedef MonoGCHandle MonoAssemblyLoadContextGCHandle;
 
-MONO_API MONO_RT_EXTERNAL_ONLY
-MonoAssembly *mono_assembly_load_full_alc (MonoAssemblyLoadContextGCHandle alc_gchandle, MonoAssemblyName *aname, const char *basedir, MonoImageOpenStatus *status);
+MONO_API MONO_RT_EXTERNAL_ONLY MonoAssembly *
+mono_assembly_load_full_alc (MonoAssemblyLoadContextGCHandle alc_gchandle, MonoAssemblyName *aname, const char *basedir, MonoImageOpenStatus *status);
 
 typedef MonoAssembly * (*MonoAssemblyPreLoadFuncV3) (MonoAssemblyLoadContextGCHandle *alc_gchandle, MonoAssemblyName *aname, char **assemblies_path, gpointer user_data, MonoError *error);
-void mono_install_assembly_preload_hook_v3 (MonoAssemblyPreLoadFuncV3 func, gpointer user_data, gboolean append);
+
+MONO_API MONO_RT_EXTERNAL_ONLY void
+mono_install_assembly_preload_hook_v3 (MonoAssemblyPreLoadFuncV3 func, gpointer user_data, gboolean append);
+
+MONO_API MONO_RT_EXTERNAL_ONLY MonoAssemblyLoadContextGCHandle
+mono_alc_default_gchandle (void);
 
```

Comment:
Would suggest naming this 'mono_get_default_alc' or 'mono_alc_get_default'.

Reply:
That's what I had initially, but it occurred to me that we have an equivalent function that gets the native ALC and I wanted to avoid confusion over what to use internally. This removes any ambiguity over what is returned.

Reply:
Would still suggest having a verb in the function name.

Reply:
Will do.